apps/freelance/src/pages/Freelancer/CreateProfile2/index.tsx

```diff
@@ -0,0 +1,214 @@
+import { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import {
+	StyledPage,
+	Form,
+	InputContainer,
+	Wrapper,
+	TextAreaContainer,
+	ButtonsContainer,
+	InputHeader,
+	InputWrapper,
+} from "./style";
+import {
+	ThemeColors,
+	ThemeBackground,
+	Dashboard,
+	StyledTitle,
+	StyledButton,
+} from "@freelance/components";
+import { ThemeProvider } from "styled-components";
+
+interface educationProps {
+	institute: string;
+	occupation: string;
+	period: string;
+}
+
+interface workExperienceProps {
+	company: string;
+	workPosition: string;
+	period: string;
+}
+
+export function CreateProfile2() {
+	const { t } = useTranslation();
+	const [educationList, setEducationList] = useState<educationProps[]>([]);
+	const [workExperienceList, setWorkExperienceList] = useState<workExperienceProps[]>([]);
+
+	const education = {
+		institute: "",
+		occupation: "",
+		period: "",
+	};
+
+	const workExperience = {
+		company: "",
+		workPosition: "",
+		period: "",
+	};
+
+	useEffect(() => {
+		setEducationList([education]);
+		setWorkExperienceList([workExperience]);
+	}, []);
+
+	const handleAddEducation = () => {
+		setEducationList(prev => [...prev, education]);
+	};
+
+	const handleRemoveEducation = (index: number) => {
+		const list = [...educationList];
+		list.splice(index, 1);
+		setEducationList(list);
+	};
+
+	const handleAddWorkExperience = () => {
+		setWorkExperienceList(prev => [...prev, workExperience]);
+	};
+
+	const handleRemoveWorkExperience = (index: number) => {
+		const list = [...workExperienceList];
+		list.splice(index, 1);
+		setWorkExperienceList(list);
+	};
+
+	return (
+		<ThemeProvider theme={ThemeColors && ThemeBackground}>
```

Comment:
it should be once in the root file, no?

Reply:
Fixed for this page. I will fix the other pages on a different branch/PR.